src/pages/trader-tools/_tools.js

```diff
@@ -127,24 +127,23 @@ const TradingTools = ({ tools }) => {
                 {tools.map((item, index) => {
                     let is_even = isIndexEven(index)
                     return (
-                        <>
-                            <ToolWrapper
-                                flex_direction={is_even ? 'row-reverse' : 'row'}
-                                key={index}
-                            >
+                        <React.Fragment key={item.image_alt}>
```

Comment:
`item.image_alt` is unique for each item? right?

Reply:
It should be ;-)